### DISCUSSION TITLE
Jeel TV

### DISCUSSION THREAD

**Discussion by genvara** [iptv-org/community#817] [Q&A]
### Channel ID

JeelDZTV.dz

### Stream URL

Channel is broadcasted here: https://jeel.dz/tv/

### Notes

A channel to add, for now it is broadcasting through the internet, but later on this month will be accessible also through satellite.

### Please confirm the following

- [X] I have read [Contributing Guide](https://github.com/iptv-org/iptv/blob/master/CONTRIBUTING.md#request-a-channel)

**Comment by R4IN2007** [comment]
https://live-api.vimeocdn.com/sessions/e07e62a0-93e6-4862-8f87-d88e09237f87/playlist/hls.m3u8?cdn=akamai&~id=player&~exp=1663790400&~sig=XwWxfqN4rwX5iSkWFzop5bG22CbttnPgKwhoJQLGZVY

**Comment by BellezaEmporium** [comment]
Added with PR iptv-org/iptv#8941

**Comment by genvara** [comment]
Looks like they do a new stream every 12 hours on vimeo and it's not a constant one stream, as the channel on the playlist now repeats the VoD, hopefully this gets fixed somehow.

This link which is currently now used in the playlist, which is also mentioned in iptv-org/iptv#7506 (http://free.fullspeed.tv/iptv-query?streaming-ip=https://vimeo.com/750785314) is now only a VoD (has a length of 12 hours) and not a livestream.

**Comment by Alstruit** [comment]
https://vimeo.com/user167465839.
I don't think fullspeed.tv has a plugin for vimeo yet.

**Comment by freearhey** [comment]
https://live-api.vimeocdn.com/sessions/e07e62a0-93e6-4862-8f87-d88e09237f87/playlist/hls.m3u8?cdn=akamai&~id=player&~exp=1663790400&~sig=XwWxfqN4rwX5iSkWFzop5bG22CbttnPgKwhoJQLGZVY (offline)
<img width="848" alt="image" src="https://user-images.githubusercontent.com/7253922/232235342-53d54110-10a4-42de-bf73-50e332245153.png">

http://free.fullspeed.tv/iptv-query?streaming-ip=https://vimeo.com/750785314 (offline)
<img width="549" alt="image" src="https://user-images.githubusercontent.com/7253922/232235358-d3b5b457-3e67-49df-94df-65a8456b557c.png">

**Comment by freearhey** [comment]
https://github.com/iptv-org/iptv/issues/19778
